Api/Model/Action/Export.php

```diff
@@ -406,17 +406,17 @@ private function getRegion(string $regionName): Region
      * @param int $maxLength
      * @return string
      */
-    private function trimChars(string $value, int $maxLength): string
+    private function trimChars(string $value = null, int $maxLength): string
     {   
-        if (strlen($value) > $maxLength) {
+        if (strlen($value ?? "") > $maxLength) {
 
             $this->logger->warning('The value is too long (magento). Trimming '.$value.' to '.$maxLength.' characters from '.strlen($value));
 
             return mb_substr($value ?? "", 0, $maxLength);
         }
         else {
 
-            return $value;
+            return $value ?? "";
```

Comment:
Do we need to convert the return value to an empty string?

Reply:
yes, based on my testing this function is expecting to return a string so if we allow null input we need to convert it